_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -24,34 +25,37 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML element][] with a [semantic][semantic role] [rowheader][] or [columnheader][] for which all of the following is true:
+This rule applies to any non-empty `th` element that is [included in the accessibility tree][], has a [semantic role][] of [rowheader][] or [columnheader][], and for which all of the following are true:
```

Comment:
"non-empty" will need some definition.
E.g. is `<th> <span></span> </th>` empty even though it has children?

Reply:
Seconded.

Reply:
Resolved in b93861a47a99b89a4b21ef64a64b4ca45ac97bf6